devices/baseEstimatorV1/README.md

```diff
@@ -18,21 +18,26 @@ This estimator uses the kinematics, IMU mounted on the head and the contact forc
     ``` sh
     gazebo -slibgazebo_yarp_clock.so
     ```
-4. Run `yarprobotinterface` with corresponding device configuration file.
-  - To run `baseEstimatorV1`,
+4. Prior to launching the `baseEstimatorV1`, launch the `wholeBodyDynamics` device.
 
     ``` sh
-     YARP_CLOCK=/clock yarprobotinterface --config launch-fbe-analogsens.xml
+     YARP_CLOCK=/clock yarprobotinterface --config launch-wholebodydynamics.xml
     ```
-    This launches both the floating base estimation device and the whole body dynamics device.
 5. Reset the offset of the FT sensors. Open a terminal and write
 
    ```
    yarp rpc /wholeBodyDynamics/rpc
    >> resetOffset all
    ```
+   Note that this step can be avoided if a flag `startWithZeroFTSensorOffsets` is set to true in the `wholeBodyDynamics` configuration file (This is applicable only in the simulation.).q
```

Comment:
typo at the end of the line?

Reply:
Fixed in https://github.com/robotology/whole-body-estimators/commit/1a36d9fb2d9eb625d56755fc2a12fe5751340d72.

> Do you think updating the CHANGELOG.md file is necessary in this case?

I think it's not necessary. What would be your opinion on this?

Reply:
You're right. I just wanted to see what you think about it.

It makes sense not to put it.. It's just a change in the documentation.